hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/spi/OzoneManagerServiceProvider.java

```diff
@@ -28,14 +26,9 @@
 public interface OzoneManagerServiceProvider {
```

Comment:
We should add lifecycle methods like stop() to gracefully allow providers and tasks to exit.